roles/installer/templates/networking/service.yaml.j2

```diff
@@ -13,7 +13,7 @@ metadata:
 {% endif %}
 spec:
   ports:
-  
+
```

Comment:
```suggestion
```

Reply:
Resolve in https://github.com/ansible/awx-operator/pull/1311